pkg/apis/config/v1alpha1/kwokctl_configuration_types.go

```diff
@@ -44,6 +44,14 @@ type KwokctlConfiguration struct {
 	Components []Component `json:"components,omitempty" patchStrategy:"merge" patchMergeKey:"name"`
 	// ComponentsPatches holds information about the components patches.
 	ComponentsPatches []ComponentPatches `json:"componentsPatches,omitempty" patchStrategy:"merge" patchMergeKey:"name"`
+	// Status holds information about the status.
+	Status KwokctlConfigurationStatus `json:"status,omitempty"`
+}
+
+// KwokctlConfigurationStatus holds information about the status.
+type KwokctlConfigurationStatus struct {
+	// Version is the version of the kwokctl.
```

Comment:
Is Version this meaning?

Version is the version of kwokctl that created this cluster.

Reply:
Yep